src/contact.ts

```diff
@@ -287,46 +295,57 @@ export class Contact implements Sayable {
   }
 
   /**
-   * get the remark for contact
+   * get the alias for contact
    */
-  public remark(): string | null
+  public alias(): string | null
   /**
-   * set the remark for contact
+   * set the alias for contact
    * @return {Promise<boolean>} A promise to the result. true for success, false for failure
    */
-  public remark(newRemark: string): Promise<boolean>
+  public alias(newAlias: string): Promise<boolean>
   /**
-   * delete the remark for a contact
+   * delete the alias for a contact
    */
-  public remark(empty: null): Promise<boolean>
+  public alias(empty: null): Promise<boolean>
 
-  public remark(newRemark?: string|null): Promise<boolean> | string | null {
-    log.silly('Contact', 'remark(%s)', newRemark || '')
+  public alias(newAlias?: string|null): Promise<boolean> | string | null {
+    log.silly('Contact', 'alias(%s)', newAlias || '')
 
-    if (newRemark === undefined) {
-      return this.obj && this.obj.remark || null
+    if (newAlias === undefined) {
+      return this.obj && this.obj.alias || null
     }
 
     return Config.puppetInstance()
-                  .contactRemark(this, newRemark)
+                  .contactAlias(this, newAlias)
                   .then(ret => {
                     if (ret) {
                       if (this.obj) {
-                        this.obj.remark = newRemark
+                        this.obj.alias = newAlias
                       } else {
-                        log.error('Contact', 'remark() without this.obj?')
+                        log.error('Contact', 'alias() without this.obj?')
                       }
                     } else {
-                      log.warn('Contact', 'remark(%s) fail', newRemark)
+                      log.warn('Contact', 'alias(%s) fail', newAlias)
                     }
                     return ret
                   })
                   .catch(e => {
-                    log.error('Contact', 'remark(%s) rejected: %s', newRemark, e.message)
+                    log.error('Contact', 'alias(%s) rejected: %s', newAlias, e.message)
                     return false // fail safe
                   })
   }
 
+  public remark(newRemark?: string|null): Promise<boolean> | string | null {
```

Comment:
should  `log.warn('Contact', 'remark(%s) DEPRECATED, use alias(%s) instead.')` be ok? I have added it under the function